### PR TITLE
5147 Typo 'bandwith' should be 'bandwidth'

### PR DESCRIPTION
**Pull request by abanink**
Updates all occurences of 'bandwith' to 'bandwidth' and 'Bandwith' to 'Bandwidth' in all .php files

After update:
```
$ find . -name "*.php" | xargs grep bandwith
$ find . -name "*.php" | xargs grep Bandwith
```

Pull request against 2018.05-rc

Note: it does not seem like there is a change in configuration keys/DB except for 'get' - so the user's settings should not be impacted - but please review.

Disclaimer: this might enable some bandwidth saver functionality that was disabled in various places due to this typo - that I did not test!

### CODE REVIEW

File: mod/update_community.php [hunk 1]
```diff
@@ -23,7 +23,7 @@ function update_community_content(App $a) {
 	$replace = "<img\${1} dst=\"\${2}\"";
 	$text = preg_replace($pattern, $replace, $text);
 
-	if (PConfig::get(local_user(), "system", "bandwith_saver")) {
+	if (PConfig::get(local_user(), "system", "bandwidth_saver")) {
```
[the remark this answers is not in the document]
Comment (MrPetovan) [reply mod/update_community.php]: You can do it like this: 

```php
PConfig::get(local_user(), "system", "bandwidth_saver", PConfig::get(local_user(), "system", "bandwith_saver"))
```

File: mod/settings.php [hunk 2]
```diff
@@ -966,7 +966,7 @@ function settings_content(App $a)
 			'$noinfo'	=> ['noinfo', L10n::t("Don't show notices"), $noinfo, ''],
 			'$infinite_scroll'	=> ['infinite_scroll', L10n::t("Infinite scroll"), $infinite_scroll, ''],
 			'$no_auto_update'	=> ['no_auto_update', L10n::t("Automatic updates only at the top of the network page"), $no_auto_update, L10n::t('When disabled, the network page is updated all the time, which could be confusing while reading.')],
-			'$bandwidth_saver' => ['bandwidth_saver', L10n::t('Bandwidth Saver Mode'), $bandwidth_saver, L10n::t('When enabled, embedded content is not displayed on automatic updates, they only show on page reload.')],
+			'$bandwidth_saver' => ['bandwidth_saver', L10n::t('Bandwith Saver Mode'), $bandwidth_saver, L10n::t('When enabled, embedded content is not displayed on automatic updates, they only show on page reload.')],
```
Comment (MrPetovan) [inline mod/settings.php]: This correction should be kept.

File: mod/admin.php [hunk 3]
```diff
@@ -1442,7 +1442,7 @@ function admin_page_site(App $a)
...
 		'$temppath'		=> ['temppath', L10n::t("Temp path"), Config::get('system','temppath'), L10n::t("If you have a restricted system where the webserver can't access the system temp path, enter another path here.")],
 		'$basepath'		=> ['basepath', L10n::t("Base path to installation"), Config::get('system','basepath'), L10n::t("If the system cannot detect the correct path to your installation, enter the correct path here. This setting should only be set if you are using a restricted system and symbolic links to your webroot.")],
-		'$proxy_disabled'	=> ['proxy_disabled', L10n::t("Disable picture proxy"), Config::get('system','proxy_disabled'), L10n::t("The picture proxy increases performance and privacy. It shouldn't be used on systems with very low bandwidth.")],
+		'$proxy_disabled'	=> ['proxy_disabled', L10n::t("Disable picture proxy"), Config::get('system','proxy_disabled'), L10n::t("The picture proxy increases performance and privacy. It shouldn't be used on systems with very low bandwith.")],
```
Comment (MrPetovan) [inline mod/admin.php]: This change should be kept.

Comment (abanink) [reply mod/admin.php]: are you sure?
In the end this string is going to be translated, so one approach could be to leave the typo in the technical string and depend on the translation to show the correct spelling.

Comment (MrPetovan) [reply mod/admin.php]: Summoning the translation master, @tobiasd

Comment (tobiasd) [reply mod/admin.php]: It should be changed. The changes will later be picked up by the translation chain.

File: mod/settings.php (hunk 2 above)
Comment (tobiasd) [reply mod/settings.php]: Jupp

File: mod/admin.php (hunk 3 above)
Comment (abanink) [reply mod/admin.php]: done

**Comment by MrPetovan** [comment]
Thank you for your contribution, sorry it took so many back and forth!

**Comment by tobiasd** [comment]
So, no conversion of old settings into new settings?

**Comment by MrPetovan** [comment]
No, there was no need. Like @abanink found out, all the `PConfig::set` calls are done with the correct spelling, while only `PConfig::get` calls were done with the incorrect spelling.

**Comment by abanink** [comment]
great
let's turn to even more important stuff now

**Comment by MrPetovan** [comment]
No contribution is insignificant.

**Comment by tobiasd** [comment]
Question is since when the setters had the correct spelling in ;-)

**Comment by MrPetovan** [comment]
Since https://github.com/friendica/friendica/commit/d9a5554543e5230828b3733f55e25304138e4601

**Comment by tobiasd** [comment]
So only a relative short time ;-) I'll add a note to the show notes about it.

**Comment by MrPetovan** [comment]
I mean, there was no setter before I added it. There was never a moment when we set the wrong spelling config key.

**Comment by MrPetovan** [comment]
I'm not sure it was done at all.

**Pull request merged**